### ISSUE TITLE
EWS as indicator for non-stationarity

### ISSUE THREAD

**Issue by drphilmarshall** [doccosmos/stationarity#1]
Hi @doccosmos !

Great hanging out with you at the AAS Hack day, Matthew - hope you had fun too :-) Are you interested in finishing the [notebook](https://github.com/doccosmos/stationarity/blob/master/Testing%20for%20stationarity%20using%20Slepian%20wavelets.ipynb) to see what the EWS looks like for our two demo light curves?

My main interest in this project is to see whether we need to make non-stationary light curves in our LSST AGN simulations. Chelsea MacLeod ( @cloumacleod ) has been playing an important role in this by working on characterizing the multi-filter stripe 82 quasar light curves, and I am pretty sure she would be interested in non-stationarity in that sample. 

Chelsea, have you ever looked for changes in DRW parameter with time? If not, you might want to follow this little project - the goal is to make a machine that can report on AGN that seem to change from one stochastic process to another.

**Comment by doccosmos** [comment]
Yes, it was nice to have time to focus to something like this. I've found a couple of related papers which give slightly more implementation details. I intend to try and do some more work on this next week (busy with other things this week). Chelsea and I have been talking about quasar variability a bit and this would certainly be interesting in that context. One thing that I've found recently is that using a Gaussian process to model the DRW seems to give better estimates for the DRW parameters that MCMC (which is never guaranteed to converge) or MLE.

**Comment by cloumacleod** [comment]
Hi Phil and Matthew, this sounds like an interesting project - thanks for inviting me!  Andy Lawrence and I have been wondering about non-stationarity in qso light curves, particularly in our PS1 nuclear hypervariables, but I haven't had much time to look into it.  So I will be interested to follow this!
